xblockgrademe/xblockgrademe.py

```diff
@@ -25,15 +25,29 @@ def workbench_scenarios():
             ('Xblock GradeMe',
              """<sequence_demo>
                     <xblockgrademe />
-                    <xblockgrademe name="My First XBlock" />
                 </sequence_demo>
              """),
         ]
 
-    name = String(
-        default='Xblock GradeMe',
+    display_name = String(
```

Comment:
This appears to be what the SDK should actually be generating; could you submit a PR to change the default?